sql/_procedures.sql

```diff
@@ -11,7 +10,6 @@ DROP TABLE IF EXISTS modifications_not_applied;
 CREATE TABLE modifications_not_applied (
 	uid 	text,
 	field 	  		text,
-	pre_corr_value 	text,
```

Comment:
Since we moved off uid matching approach, the `pre_corr_value` for each individual record is no longer checked and so it is not applied. Instead, for all records that has the field matched with the `old_value` they values are going to be modified at once. My thinking is it have become redundant and its values are always null anyway.